test/NonSiloTests/Orleans.NonSiloTests.csproj

```diff
@@ -168,6 +213,10 @@
       <Project>{6ff2004c-cdf8-479c-bf27-c6bfe8ef93e0}</Project>
       <Name>OrleansRuntime</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\OrleansSQLUtils\OrleansSQLUtils.csproj">
+      <Project>{47e9ae37-4eae-48d5-a778-2c80e9a6375f}</Project>
+      <Name>OrleansSQLUtils</Name>
+    </ProjectReference>
```

Comment:
same as before, this test project should be able to test the Orleans core without any extensibility providers